Add detection rule for command-line download instructions

morriscode opened the pull request:
# Description

This rule detects messages that contain PowerShell commands or system utilities followed by domain names, indicating possible malicious activity.

# Associated samples
- 4fcbf0f4c9e577ec9e0ea4a7140ecbac11a9081cb8886a0e060c0360797ddd7f


## Associated hunts


- [Hunt 1
](https://platform.sublime.security/messages/hunt?huntId=019b146e-8910-731f-8d4c-b513a4641b46)

morriscode committed:
Add detection rule for command-line download instructions

This rule detects messages that contain PowerShell commands or system utilities followed by domain names, indicating possible malicious activity.

Commit:
Auto add rule ID

MSAdministrator reviewed (approved):
Looks good to me

MSAdministrator commented:
I ran hunts in several environments and all matches resulted in a `benign` attack score. Since this is an ASR rule, I think we are good to move forward.

aidenmitchell committed:
Merge branch 'main' into morriscode-patch-8

MSAdministrator committed:
Merge branch 'main' into morriscode-patch-8

MSAdministrator committed:
Update body_command_line_download_instructions.yml

morriscode committed:
Update body_command_line_download_instructions.yml

pulling ftp|tftp protos

morriscode committed:
Update body_command_line_download_instructions.yml

morriscode committed:
Refactor trusted sender domain checks in YAML

Updated conditions for trusted sender domains // added boundaries around the weblink pattern post invocation

Commit:
Auto-format MQL and add rule IDs

MSAdministrator committed:
Merge branch 'main' into morriscode-patch-8

morriscode committed:
Fix regex pattern for command line download detection

morriscode committed:
Exclude newsletters and digests from detection rules

Added a condition to exclude newsletters and digests from detection.

morriscode committed:
Update body_command_line_download_instructions.yml

MSAdministrator commented:
Was reviewing this and wanted to share a [multi-hunt](https://hunt.limeseed.email/hunts/040d9526-17a0-4bb5-8fef-f9da6ae80c58)
